datalad/support/gitrepo.py

```diff
@@ -1385,6 +1385,18 @@ def get_tracking_branch(self, branch=None):
 
         return track_remote, track_branch
 
+    @property
+    def count_objects(self):
+        """return dictionary with count, size(in KiB) information of git objects
+        """
+
+        count_cmd = ['git', 'count-objects', '-v']
+        count_str, err = self._git_custom_command('', count_cmd)
+        count = {key: int(value)
+                 for key, value in [item.split(': ')
+                                    for item in count_str.split('\n')
+                                    if len(item.split(': ')) == 2]}
```

Comment:
so `if not item.strip()` is not sufficient? but ok -- I guess better be safe than sorry